spotlight-client/src/contentModels/RacialDisparitiesNarrative.ts

```diff
@@ -86,16 +91,37 @@ function getSentencingMetrics(
   };
 }
 
+type ChartLabels = {
+  totalPopulation: string;
+  totalSentenced: string;
+};
+
 type ConstructorOpts = {
   tenantId: TenantId;
   defaultCategory?: RaceIdentifier;
   defaultSupervisionType?: SupervisionType;
 };
 
+/**
+ * Data model for the racial disparities narrative, which handles computation
+ * as well as fetching. The preferred method for instantiating this class is
+ * to use the static `build` method.
+ * @example
+ *
+ * ```js
+ * const narrative = RacialDisparitiesNarrative.build(props);
+ * ```
+ */
 export default class RacialDisparitiesNarrative {
   // metadata
   readonly title = "Racial Disparities";
 
+  // TODO: customize text?
```

Comment:
Did you want to resolve this TODO in this PR?

Reply:
yeah that comment was a straggler, it's resolved now